plan/physical_plans.go

```diff
@@ -819,34 +855,43 @@ func (p *PhysicalMergeJoin) Copy() PhysicalPlan {
 
 // MarshalJSON implements json.Marshaler interface.
 func (p *PhysicalHashJoin) MarshalJSON() ([]byte, error) {
-	leftChild := p.children[0].(PhysicalPlan)
-	rightChild := p.children[1].(PhysicalPlan)
-	eqConds, err := json.Marshal(p.EqualConditions)
-	if err != nil {
-		return nil, errors.Trace(err)
+
+	buffer := bytes.NewBufferString("{")
+	buffer.WriteString(fmt.Sprintf("\"small table\":\"%s\",", p.children[p.SmallTable].ID()))
```

Comment:
For hash join, "small table" indicates which table executor will use to build hash table ? Maybe we only need to print the "small table" information